src/index.tsx

```diff
@@ -15,11 +15,50 @@ import React, {
 // eslint-disable-next-line @typescript-eslint/ban-types
 type ComponentDidCatch = ComponentLifecycle<{}, {}>["componentDidCatch"];
 
+type ErrorType = WrappedError | Error | undefined;
```

Comment:
```suggestion
```

Reply:
My reasoning behind this was to more clearly annotate the types for users, so that they know they can import `WrappedError` and use `instanceof` to check if `.originalData` will be present.

I think we should still keep the `| undefined` because when using [TS in strict mode](https://www.typescriptlang.org/tsconfig#strictNullChecks), `undefined` and `null` are no longer ignored by TS, and this could cause unexpected errors for users of the package with TS in strict mode (`error === undefined` will be interpreted as `never` as TS doesn't know this might be `undefined`).

Reply:
Yes on the call sites where undefined is necessary -- I was thinking we could inline these.

My inclination is to keep `WrappedError` private to the package for now. The property accessor is a nice convenience for local debugging.